script2.js

```diff
@@ -0,0 +1,94 @@
+const API = 'https://pokeapi.co/api/v2/pokemon/'
+const wrapper = document.querySelector('wrapper')
+const main = document.querySelector('main')
+const pokemonOne = document.querySelector('.pokemonOne')
+const backCards = Array.from(document.querySelectorAll('.backCard'))
+const next = document.querySelector('.next')
+const previous = document.querySelector('.previous')
+const page = document.querySelector('.currentPage');
+let localDataBase = ['empty'];
+let pokemonsFetchesSoFar = 0;
+let currentPage = 0;
+let c = pokemonsFetchesSoFar; // counter
+let allPages = [];
+
+async function fetchPokemons() {
+    let pokemonsFetched = [];
+
+    for (let i = 0; i < 3; i++) {
+        c++
+        pokemonsFetched.push(await (await fetch(API + c)).json());
+    }
+    localDataBase.push(pokemonsFetched);
+    return pokemonsFetched
+}
```

Comment:
Zmienić nazwę parametru `fn` na bardziej opisowy
```suggestion
async function domUpdate(callback, direction) {
```